dv/uvm/core_ibex/sim_makefrag_gen.py

```diff
@@ -79,7 +94,10 @@ def main():
         'Generates a makefile fragment for use with the Ibex DV makefile that '
         'sets up sim specific variables'))
 
-    sim_fns = {'vcs': gen_vcs_makefrag, 'riviera': gen_riviera_makefrag, 'xlm': gen_xlm_makefrag}
+    sim_fns = {'vcs'    : gen_vcs_makefrag,
+               'riviera': gen_riviera_makefrag,
+               'xlm'    : gen_xlm_makefrag,
+               'questa' : gen_questa_makefrag}
```

Comment:
Sorry, I realise this is really nitty, but I think PEP8 standard style doesn't allow tabulating dicts like this. Could you drop the extra spaces?

Reply:
If you run flake8 on the file, you should hopefully get no errors. We've got that check in OpenTitan CI, but apparently not in Ibex.

Reply:
I've fixed it and run it through flake8 (didn't know that tool)